tokio/src/macros/try_join.rs

```diff
@@ -59,6 +59,44 @@
 ///     }
 /// }
 /// ```
+///
+/// Returning on the first error.
```

Comment:
I think this is a poor description of the example.
```suggestion
/// Using `try_join!` with spawned tasks.
```

Reply:
Thanks for your review!  💚 💙 💜 💛 ❤️ Addressed!